unit_testing/test_math.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+# pytest unit tests for spinalcordtoolbox.math
+
+
+from __future__ import absolute_import
+import sys
+import os
+import pytest
+import math
+import numpy as np
+from skimage.morphology import erosion, dilation, disk, ball, square, cube
```

Comment:
> +./unit_testing/test_math.py:12: 'skimage.morphology.erosion' imported but unused
> +./unit_testing/test_math.py:12: 'skimage.morphology.dilation' imported but unused
> +./unit_testing/test_math.py:12: 'skimage.morphology.disk' imported but unused
> +./unit_testing/test_math.py:12: 'skimage.morphology.ball' imported but unused
> +./unit_testing/test_math.py:12: 'skimage.morphology.square' imported but unused
> +./unit_testing/test_math.py:12: 'skimage.morphology.cube' imported but unused

Reply:
fixed in 5a8e88d29a694198f4e46b7279ef1d625e9bb50f

Reply:
oops scrap that-- good one c69ec1e14f6b88c487823b9ff2f28c8e10dec776